e2e/test/MessageE2ETests.cs

```diff
@@ -597,6 +598,54 @@ public async Task Message_NoTimeoutPassed()
             await DefaultTimeout().ConfigureAwait(false);
         }
 
+#if !NET451
```

Comment:
For some reason, I couldn't access the event hub libraries needed to verify that a message was received. It seems like other tests in this suite do not have this issue. I could use some help understanding this issue.